### PR TITLE
CanvasRenderingContext2d createConicGradient

### PR DESCRIPTION
**Pull request by Rumyra**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes

Adding data for `CanvasRenderingContext2d.createConicGradient()`

- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)

https://bugs.chromium.org/p/chromium/issues/detail?id=1097034&q=1097034&can=2
https://bugzilla.mozilla.org/show_bug.cgi?id=1627014

- [x] Data: if you tested something, describe how you tested with details like browser and version

Works in both Chrome & FF for me (FF 86 with canvas.createConicGradient.enabled. Does not work in latest Opera or Safari desktop

- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)

No errors

- [x] Link to related issues or pull requests, if any

https://github.com/mdn/content/issues/1724

### CODE REVIEW

**Review by ddbeck — CHANGES_REQUESTED** [review]
[Bug 1097034](https://bugs.chromium.org/p/chromium/issues/detail?id=1097034) says this is implemented behind a flag in Chrome too (I couldn't get it to work in Chrome by default, too). Could you update this with the flag data? Thank you!

**Comment by Rumyra** [comment]
Oh I'm so sorry I missed that 😬 I've added the flag (still checking syntax every time ☺️ hope this is correct)

**Comment by chandonglee** [comment]
no problem

File: api/CanvasRenderingContext2D.json [hunk 1]
```diff
@@ -479,6 +479,78 @@
...
+      "createConicGradient": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CanvasRenderingContext2D/createConicGradient",
+          "support": {
+            "chrome": {
+              "version_added": "86",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "new-canvas-2d-api"
```
Comment (ddbeck) [inline api/CanvasRenderingContext2D.json:491]:
```suggestion
                  "name": "#new-canvas-2d-api"
```

File: api/CanvasRenderingContext2D.json [hunk 2]
```diff
@@ -479,6 +479,78 @@
...
+                }
+              ]
+            },
+            "chrome_android": {
+              "version_added": "86",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "new-canvas-2d-api"
```
Comment (ddbeck) [inline api/CanvasRenderingContext2D.json:500]:
```suggestion
                  "name": "#new-canvas-2d-api"
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
Looks really good, but there's one tiny nit for consistency. Thank you!

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**